fw/Core/Hitcon/Hitcon.cpp

```diff
@@ -56,7 +57,7 @@ void hitcon_run() {
   display_init();
   g_noise_source.Init();
   g_entropy_hub.Init();
-  g_hash_service.Init();
+  // g_hash_service.Init();
```

Comment:
Revert?

Reply:
This was intentional. Will uncomment after testing.